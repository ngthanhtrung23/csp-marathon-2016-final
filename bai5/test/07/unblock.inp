1..2..
1..2..
00.3.4
5553.4
.677.4
.688..
