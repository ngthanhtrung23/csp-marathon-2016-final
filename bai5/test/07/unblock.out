15
4 U 1
7 R 2
8 R 2
3 D 2
0 R 2
5 R 2
6 U 4
0 L 1
5 L 2
3 U 2
7 L 3
3 D 1
8 L 1
4 D 2
0 R 3

